tests/components/switcher_kis/__init__.py

```diff
@@ -1,14 +1,27 @@
 """Test cases and object for the Switcher integration tests."""
 
-from homeassistant.components.switcher_kis.const import DOMAIN
+from homeassistant.components.switcher_kis.const import (
+    CONF_TOKEN,
+    CONF_USERNAME,
+    DOMAIN,
+)
 from homeassistant.core import HomeAssistant
 
 from tests.common import MockConfigEntry
 
 
-async def init_integration(hass: HomeAssistant) -> MockConfigEntry:
+async def init_integration(
+    hass: HomeAssistant, username: str | None = None, token: str | None = None
+) -> MockConfigEntry:
     """Set up the Switcher integration in Home Assistant."""
-    entry = MockConfigEntry(domain=DOMAIN, data={}, unique_id=DOMAIN)
+    entry = MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_USERNAME: username,
+            CONF_TOKEN: token,
+        },
```

Comment:
Add the `username` and `token` to the data only if they are not `none`. This is not a valid mocking as existing entries will not have these fields at all and new entries will have them with a value.
Example:
https://github.com/home-assistant/core/blob/41b129e990d9c2993dab3e34e9df37faa1059856/tests/components/shelly/__init__.py#L50-L51